src/mlpack/core/data/feature_selection.hpp

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file feature_selection_imp.hpp
+ * @author Jeffin Sam
+ *
+ * Feature selction based on variance thresholding.
+ * Motivated by the idea that low variance features contain less
+ * information.
+ * Calculate varience of each feature, then drop features 
+ * with variance below some threshold
```

Comment:
Can we reference a paper here?

Reply:
I added this[ paper](https://ieeexplore.ieee.org/document/4959366/references#references) as reference, but also found this [paper](http://cse.vnit.ac.in/people/narendraschaudhari/wp-content/uploads/sites/15/2014/10/054-NaiveBayesian-Yilin.pdf) interesting, but couldn't find out where was the latter one published.

Looking forward for your take on which one should i add.